caddyhttp/fastcgi/fastcgi.go

```diff
@@ -12,10 +12,29 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+	"sync"
 
 	"github.com/mholt/caddy/caddyhttp/httpserver"
 )
 
+// persistent fastcgi connections
+
+type serialClient struct {
+	// for read/write serialisation
+	sync.Mutex
+	backend *FCGIClient
+}
+type concurrentPersistentConnectionsMap struct {
+	// for thread safe acces to the map
+	sync.Mutex
+	clientMap map[string]*serialClient
+}
+
+var persistentConnections *concurrentPersistentConnectionsMap = &(concurrentPersistentConnectionsMap{clientMap: make(map[string]*serialClient)})
```

Comment:
You can remove the type declaration here (`*concurrentPersistentConnectionsMap) - see the lint warnings. It will shorten the line quite a bit :)